### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by rgnarok** [VAYUZ-VLCC/vlcc-api-issues#118]
No notification when i cancel the notification also the text which is shared by VLCC team is not getting reflected

**Comment by Shaktivayuz** [comment]
You will not get any notification when you disable notification .All notification text coming from backend

**Comment by Shaktivayuz** [comment]
this is working

**Comment by rgnarok** [comment]
Text is still not working @PM-Vallesoft  please look itno this the text of the notification for customer is coming which is for the expert plus the text which is added on the sheet shared earlier by us it is not as per what VLCC team finalised. 
Check the latest apk attached
https://drive.google.com/file/d/1mleQ7rdfqwMLH0K6bmYQ6lc7JAUInkpt/view?ts=61d3cc67
Sheet for triggers: https://docs.google.com/spreadsheets/d/1Rw5A2cg6UuiJClYRk9aylpyjxLzLvY2r29qkcegZEsM/edit?usp=sharing
@PankajBhutani @ibsingh13

**Comment by PM-Vallesoft** [comment]
> Text is still not working @PM-Vallesoft please look itno this the text of the notification for customer is coming which is for the expert plus the text which is added on the sheet shared earlier by us it is not as per what VLCC team finalised. Check the latest apk attached https://drive.google.com/file/d/1mleQ7rdfqwMLH0K6bmYQ6lc7JAUInkpt/view?ts=61d3cc67 Sheet for triggers: https://docs.google.com/spreadsheets/d/1Rw5A2cg6UuiJClYRk9aylpyjxLzLvY2r29qkcegZEsM/edit?usp=sharing @PankajBhutani @ibsingh13

Yes same for expert and client since few no like 8951900164 are in both Expert and client db...
Test as per vlcc team sill needs to be changed..... from my end and will do it shortly.

**Comment by rgnarok** [comment]
@PM-Vallesoft i am testing for the number 8377047399 which is nit an expert but still the above issue is there. Please check this at your end too.
@PankajBhutani
